### DISCUSSION TITLE
Custom Images in MS Visio

### DISCUSSION THREAD

**Discussion by TDills** [fernandreu/office-ribbonx-editor#247] [Q&A]
Adding custom images to a Visio file fail when activating the custom tab.
The error code provide is:
Custom UI RUntime Error in "...."
No loadImage callback was specified to load the image with id "..."

Running the exact same code in Excel documents work beautifully (without manually specifying loadimage callback), so it seems to be specific to Visio.

**To Reproduce**
1. Add a custom image to a button (in MS Visio!)
2. Activate the Tab in question

**Expected behavior**
The custom image should appear as an icon in the ribbon, just like it does in Excel.

**Screenshots**
See error code

**Additional context**
Ribbon Code:
<customUI xmlns="http://schemas.microsoft.com/office/2009/07/customui">    
  
  
  
  

  <ribbon>
    <tabs>
      <tab id="customTab1" label="CustomTab" insertAfterMso="TabHome">
    
        <group idMso="GroupArrange" />
          
        <group id="CustomGroup10" label="Test Tab">
            <button id="customButton101" label="Custom Image" size="large" 
                onAction="runMacro" image="VisioIcon_GW3" />
        </group>              
      </tab>
    </tabs>
  </ribbon>
</customUI>

**Comment by fernandreu** [comment]
Sorry, this issue is not due to the tool itself, so I cannot help. I will move it to discussions in case somebody else can
